doc/usecases/visitor_chat.md

```diff
@@ -0,0 +1,26 @@
+# Use Case: Visitor chat
+
+## Summary
+
+- **Scope:** Map View
+- **Level:** User Goal
+- **Actors:** 
+  - Owner: App User who owns the map
+  - Visitor: App User who visits the map
+- **Brief:** The user can interact with a visitor chat.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  The visitor has opened the app and selected a map.
```

Comment:
Should we specify here if a user is logged in or an anonymous user?

Reply:
I don't think we should avoid not-logged-in users to participate in any way, the risk for spam is too high.

Reply:
Imho they also shouldn't even be allowed to read the chat. I think this is against the expectations, even if someone creates a public map.

@aidnurs great questions, thanks for the suggestion.